src/common/models/Projection.ts

```diff
@@ -201,21 +216,38 @@ export class Projection {
       /*includeTrailingZeros=*/ false,
     );
 
-    const cumulativeConfirmedCases = this.smoothCumulatives(
+    this.rawDailyCases = this.deltasFromCumulatives(
       this.fillLeadingNullsWithZeros(
         actualTimeseries.map(row => row && row.cumulativeConfirmedCases),
       ),
     );
-    this.smoothedDailyCases = this.smoothWithRollingAverage(
-      this.deltasFromCumulatives(cumulativeConfirmedCases),
+    this.smoothedDailyCases = this.smoothWithRollingAverage(this.rawDailyCases);
+
+    this.rawDailyDeaths = this.deltasFromCumulatives(
+      this.fillLeadingNullsWithZeros(
+        actualTimeseries.map(row => row && row.cumulativeDeaths),
+      ),
+    );
+    this.smoothedDailyDeaths = this.smoothWithRollingAverage(
+      this.rawDailyDeaths,
+    );
+
+    this.rawHospitalizations = actualTimeseries.map(
+      row => row && row?.hospitalBeds?.currentUsageCovid,
```

Comment:
Annoyingly the `?.` operator defaults to `undefined` instead of null which doesn't match our type definition (`Array<number | null>`) and may break some of our assumptions in other parts of the code.  I _think_ you can do `row && row.hospitalBeds.currentUsageCovid` instead.